src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        int size = 6;
+        FigureDrawable[] figures = new FigureDrawable[size];
+
+        for (int i = 0; i < size / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (int i = size / 2; i < size; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
+        }
+
+        for (FigureDrawable figure: figures) {
+            figure.getInfo();
```

Comment:
It seems like you are calling `getInfo()` method which is expected to return some information about the figure, but you're not using its return value. If `getInfo()` is supposed to print the information to the console, then it should be renamed to reflect that action, such as `printInfo()`. Always ensure method names clearly represent their behavior.